src/fights.ts

```diff
@@ -261,29 +261,34 @@ function getEmbezzlerFight(): EmbezzlerFight | null {
     if (fight.available()) return fight;
   }
   const potential = embezzlerCount();
-  if (
-    potential > 0 &&
-    get("_genieFightsUsed") < 3 &&
-    userConfirm(
-      `Garbo has detected you have ${potential} potential ways to copy an Embezzler, but no way to start a fight with one. Should we wish for an Embezzler?`
-    )
-  ) {
-    return new EmbezzlerFight(
-      "Pocket Wish",
-      () => false,
-      () => 0,
-      () => {
-        retrieveItem($item`pocket wish`);
-        visitUrl(`inv_use.php?pwd=${myHash()}&which=3&whichitem=9537`, false, true);
-        visitUrl(
-          "choice.php?pwd&whichchoice=1267&option=1&wish=to fight a Knob Goblin Embezzler ",
-          true,
-          true
-        );
-        visitUrl("main.php", false);
-        runCombat();
-      }
-    );
+  if (potential > 0 && get("_genieFightsUsed") < 3) {
```

Comment:
I'd say just always print them before the genie check so they end up in the log if someone has already used their wishes.